content/html/encyclopedia/tables.md

```diff
@@ -0,0 +1,108 @@
+---
+Subjects:
+  - "HTML/CSS"
```

Comment:
"HTML & CSS" is how it appears at https://author.codecademy.com/browse/categories